### PR TITLE
Design of events page

### PR DESCRIPTION
**Pull request by llincker**
**Design update, using semantic-ui.**
Improves the appearance of the Events page, which consists in a succession of events : 

To do : ease the access to one particular event (-> accordion with the titles or list of the titles with routers?). Is the number of the event needed ?

The drop-down (to choose the products) still needs some improvements, for example allowing to search for the product you want by beginning to type its name.

### CODE REVIEW

File: src/routes/Login.vue [hunk 1]
```diff
@@ -1,29 +1,29 @@
 <template>
```
Comment (llincker) [inline src/routes/Login.vue:1]: I've only re-indented this file, nothing has changed.

**Comment by febeck** [comment]
Qu'est-ce que c'est ce truc chelou?

captura de tela 2017-03-09 as 15 13 59

**Comment by llincker** [comment]
C'est la partie sur laquelle il faut encore que je bosse, la sélection des produits.

**Comment by febeck** [comment]
D'acc! Je n'ai pas regardé le code, mais cool pour le design du reste de la page du coup!

**Review by LoicBer — APPROVED** [review]

**Comment by LoicBer** [comment]
déso

**Pull request merged**